### PR TITLE
Finally switch from request to node-fetch

### PR DESCRIPTION
**Pull request by TheMonDon**
**Please describe the changes this PR makes and why it should be merged:**
Request has been deprecated for over a year.

Also removed the "URL Checker" thing, didn't work, seemed useless.
Also removed instructions.

**Status**

- [X] Code changes have been tested against the Discord API and the discord.js wrapper, or there are no code changes
- [X] Documentation has been added/modified, or there is nothing to change (docs/mods.json)

**Semantic versioning classification:**

- [ ] This PR changes DBM's interface (methods or parameters added to default methods)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: actions/store_html_from_webpage_MOD.js [hunk 1]
```diff
@@ -17,98 +16,54 @@ module.exports = {
...
+        const html = response.body
+        this.storeValue(html.trim(), storage, varName, cache)
+        this.callNextAction(cache)
+      } catch (err) {
+        console.error(err)
       }
-    } catch (error) {
-      console.error(`API Things:  Error: ${error.stack}` ? error.stack : error)
+    } else {
+      throw Error(`HTML Parser - URL [${url}] Is Not Valid`)
```
Comment (almeidx) [inline actions/store_html_from_webpage_MOD.js:66]:
```suggestion
      console.error(`HTML Parser - URL [${url}] Is Not Valid`)
```

**Review by almeidx — CHANGES_REQUESTED** [review]
Any reason not to use `node-fetch` instead of `got`?

**Comment by TheMonDon** [comment]
> Any reason not to use `node-fetch` instead of `got`?

Updated more, maintained more, and seems like a better thing overall.

**Comment by TheMonDon** [comment]
ig node-fetch isn't so bad, still don't like u tho

File: actions/store_html_from_webpage_MOD.js [hunk 2]
```diff
@@ -17,98 +16,54 @@ module.exports = {
...
 
-          this.storeValue(html.trim(), storage, varName, cache)
-          this.callNextAction(cache)
-        })
-      } else {
-        throw Error(`HTML Parser - URL [${url}] Is Not Valid`)
+    if (Mods.checkURL(url)) {
+      try {
+        const response = await fetch(url)
+        const html = await response.text()
```
Comment (almeidx) [inline actions/store_html_from_webpage_MOD.js:59]:
```suggestion
        const html = await fetch(url).then((r) => r.text())
```

**Review by almeidx — CHANGES_REQUESTED** [review]

**Review by almeidx — APPROVED** [review]

**Pull request merged**